pkg/controller/cluster/main-controller.go

```
@@ -387,19 +389,23 @@ func (c *Controller) syncHandler(key string) error {
 		if !(mi.ExternalCert() && mi.ExternalClientCert() && mi.KESExternalCert()) {
 			msg := "Please provide certificate secrets for MinIO and KES, since automatic TLS is disabled"
 			klog.V(2).Infof(msg)
-			return fmt.Errorf(msg)
+			var err2 error
+			if mi, err2 = c.updateTenantStatus(ctx, mi, msg, 0); err2 != nil {
+				klog.V(2).Infof(err2.Error())
+			}
+			return nil
```

Why not return the error?

---

Same answer as above